apps/web/src/features/search/components/filters/date-range-picker.tsx

```
@@ -0,0 +1,239 @@
+import { useState } from 'react';
+import {
+  Calendar,
+  CalendarDays,
+  ChevronLeft,
```

ESLint

[error] 'ChevronLeft' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)